docs/sphinx_source/README.md

```diff
@@ -0,0 +1,223 @@
+
+***Work in progress***
+
+* [2D](https://www.osapublishing.org/oe/fulltext.cfm?uri=oe-23-9-11394&id=315599) DPC
+* [3D](https://www.osapublishing.org/boe/fulltext.cfm?uri=boe-7-10-3940&id=349951) DPC
+
+
+## Fluorescence microscopy
+
+### Widefield deconvolution microscopy
+__fluorescence density__ from a widefield volumetric fluorescence acquisition.
+
+<details>
+<summary> Swedlow 2013 </summary>
+<pre><code>
+@article{Swedlow:13,
+author = {Swedlow, John R.},
+journal = {Methods Cell Biol.},
+title = {Quantitative fluorescence microscopy and image deconvolution},
+year = {2013},
+volume = {114},
+pages = {407--26},
+doi = {10.1016/B978-0-12-407761-4.00017-8}
+}
+</code></pre>
+</details>
+
+### Oblique plane light-sheet microscopy
+__fluorescence density__ from oblique plane light-sheet microscopy.
+
+<details>
+<summary> Ivanov, Hirata-Miyasaki, Chandler et al. 2024 </summary>
+<pre><code>
+@article{ivanov_2024,
+author = {Ivanov, Ivan E. and Hirata-Miyasaki, Eduardo and Chandler, Talon and Cheloor-Kovilakam, Rasmi and Liu, Ziwen and Pradeep, Soorya and Liu, Chad and Bhave, Madhura and Khadka, Sudip and Arias, Carolina and Leonetti, Manuel D. and Huang, Bo and Mehta, Shalin B.},
+title = {Mantis: High-throughput 4D imaging and analysis of the molecular and physical architecture of cells},
+journal = {PNAS Nexus},
+volume = {3},
+number = {9},
+pages = {pgae323},
+year = {2024},
+doi = {10.1093/pnasnexus/pgae323}
+</code></pre>
+</details>
+
+
+## Citation
+
+Please cite this repository, along with the relevant publications and preprints, if you use or adapt this code. The citation information can be found by clicking "Cite this repository" button in the About section in the right sidebar.
+
+## Installation
+
+Create a virtual environment:
+
+```sh
+conda create -y -n waveorder python=3.12
+conda activate waveorder
+```
+
+(Option 1) If you are familiar with `waveorder`'s API and want to use it as a library, install a stable version of `waveorder` from PyPI:
+
+```sh
+pip install waveorder
+```
+
+(Option 2) If you are new to `waveorder` and want to run demos to become familiar with its features, install the latest version of `waveorder` from GitHub with all visualization dependencies (napari, jupyter), clone the repository, and run an example script:
+```sh
+pip install "git+https://github.com/mehta-lab/waveorder.git@main#egg=waveorder[all]"
+git clone https://github.com/mehta-lab/waveorder.git
+python waveorder/docs/examples/models/phase_thick_3d.py
+```
+
+(M1 users) `pytorch` has [incomplete GPU support](https://github.com/pytorch/pytorch/issues/77764),
+so please use `export PYTORCH_ENABLE_MPS_FALLBACK=1`
+to allow some operators to fallback to CPU if you plan to use GPU acceleration for polarization reconstruction.
+
+
+## Examples
+The [examples](https://github.com/mehta-lab/waveorder/tree/main/docs/examples) illustrate simulations and reconstruction for 2D QLIPP, 3D phase from brightfield, and 2D/3D PTI methods.
```

Comment:
Q - Clarification: Should this be directing users to the directory on Github or to the Examples page on the docs site?

Reply:
This `README.md` now only lives in the root of the repo, so I think linking to the github examples makes more sense now.